src/endless/WMI.cpp

```diff
@@ -102,7 +102,7 @@ BOOL WMI::IsBitlockedDrive(const CString & drive)
     const CString objectPath(L"ROOT\\CIMV2\\Security\\MicrosoftVolumeEncryption");
     CString bitlockerQuery;
 
-    bitlockerQuery.Format(L"Select * from Win32_EncryptableVolume where ProtectionStatus != 0 and DriveLetter = '%ls'", drive);
+    bitlockerQuery.Format(L"Select * from Win32_EncryptableVolume where ConversionStatus != 0 and DriveLetter = '%ls'", drive);
```

Comment:
Well spotted. Can we add a link to https://msdn.microsoft.com/en-us/library/windows/desktop/aa376483(v=vs.85).aspx in this function? And yes, let's wait and see what Beatriz comes back with.